global-news-app/index.html

```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap">
```

Comment:
i would recommend to self host the fonts. you can use the `@font-face` rule in your global styles.

for example: 

`@font-face {
  font-family: 'Font Name';
  src: url('path/to/font.woff2') format('woff2')
}`